ci-operator/config/openshift/velero/openshift-velero-oadp-1.2.yaml

```diff
@@ -0,0 +1,44 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
Per https://github.com/vmware-tanzu/velero/blob/73f17404078f0247c5d27af3890d8f33b598ff6c/Dockerfile#L16
This needs to be 1.19

Reply:
```suggestion
    tag: golang-1.19
```